public/app/features/integrated-alerting/components/Alerts/Alerts.tsx

```diff
@@ -1,4 +1,128 @@
-import React, { FC } from 'react';
-import { AlertsTable } from './AlertsTable';
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { useStyles, useTheme } from '@grafana/ui';
+import { logger } from '@percona/platform-core';
+import { Column } from 'react-table';
+import { cx } from 'emotion';
+import { Table } from '../Table/Table';
+import { Messages } from '../../IntegratedAlerting.messages';
+import { getStyles } from './Alerts.styles';
+import { Alert, AlertStatus } from './Alerts.types';
+import { formatAlerts, getSeverityColors } from './Alerts.utils';
+import { AlertsService } from './Alerts.service';
+import { AlertRuleSeverity } from '../AlertRules/AlertRules.types';
+import { AlertsActions } from './AlertsActions';
 
-export const Alerts: FC = () => <AlertsTable />;
+const { noData, columns } = Messages.alerts.table;
```

Comment:
Why this big changes to `Alerts` in this PR about backups? 🤔

Reply:
Same reason as above ;)